pom.xml

```diff
@@ -1225,7 +1225,7 @@
             <dependency>
                 <groupId>com.microsoft.sqlserver</groupId>
                 <artifactId>mssql-jdbc</artifactId>
-                <version>10.2.0.jre11</version>
+                <version>11.2.0.jre17</version>
```

Comment:
It's a very recent release. IMO we should wait for at least 1 more release to make sure bugs are caught. In the past we've had some cases where regressions were found due to driver bugs.

Also Java is backward compatible so a JDK 11 lib (which doesn't do illegal things) should keep working.

Reply:
Sure.. we can wait.

Also I realize that the JDK 11 version will continue to work but the 17 version will use better optimized bytecode and hence be better to use. We do require Java 17 after all ;-)

Reply:
I think so.

Reply:
Done